src/coreclr/vm/riscv64/stubs.cpp

```diff
@@ -1104,117 +1104,6 @@ void StubLinkerCPU::EmitMovConstant(IntReg reg, UINT64 imm)
-    // prolog and epilog generation as well as unwindinfo generation.
-    _ASSERTE((totalPaddedFrameSize <= 504) && "NYI:RISCV64 Implement StubLinker prologs with larger than 504 bytes of frame size");
-    if (totalPaddedFrameSize > 504)
-        COMPlusThrow(kNotSupportedException);
-
-    // Here is how the stack would look like (Stack grows up)
-    // [Low Address]
-    //            +------------+
-    //      SP -> |            | <-+
-    //            :            :   | Stack Frame, (i.e outgoing arguments) including padding
-    //            |            | <-+
-    //            +------------+
-    //            | FP         |
-    //            +------------+
-    //            | RA         |
-    //            +------------+
-    //            | F10        | <-+
-    //            +------------+   |
-    //            :            :   | Fp Args
-    //            +------------+   |
-    //            | F17        | <-+
-    //            +------------+
-    //            | X10        | <-+
-    //            +------------+   |
-    //            :            :   | Int Args
-    //            +------------+   |
-    //            | X17        | <-+
-    //            +------------+
-    //  Old SP -> |[Stack Args]|
-    // [High Address]
-
-    // Regarding the order of operations in the prolog and epilog;
-    // If the prolog and the epilog matches each other we can simplify emitting the unwind codes and save a few
-    // bytes of unwind codes by making prolog and epilog share the same unwind codes.
-    // In order to do that we need to make the epilog be the reverse of the prolog.
-    // But we wouldn't want to add restoring of the argument registers as that's completely unnecessary.
-    // Besides, saving argument registers cannot be expressed by the unwind code encodings.
-    // So, we'll push saving the argument registers to the very last in the prolog, skip restoring it in epilog,
-    // and also skip reporting it to the OS.
-    //
-    // Another bit that we can save is resetting the frame pointer.
-    // This is not necessary when the SP doesn't get modified beyond prolog and epilog. (i.e no alloca/localloc)
-    // And in that case we don't need to report setting up the FP either.
-
-    // 1. Relocate SP
-    EmitSubImm(RegSp, RegSp, totalPaddedFrameSize);
-
-    unsigned cbOffset = 2 * sizeof(void*) + cbStackSpace; // 2 is for fp, ra
-
-    // 2. Store FP/RA
-    EmitStore(RegFp, RegSp, cbStackSpace);
-    EmitStore(RegRa, RegSp, cbStackSpace + sizeof(void*));
-
-    // 3. Set the frame pointer to the Canonical Frame Address or CFA, which is the stack pointer value on entry to the
-    // current procedure
-    EmitAddImm(RegFp, RegSp, totalPaddedFrameSize);
-
-    // 4. Store floating point argument registers
-    _ASSERTE(cFpRegArgs <= 8);
-    for (unsigned short i = 0; i < cFpRegArgs; i++)
-        EmitStore(FloatReg(i + 10), RegSp, cbOffset + i * sizeof(void*));
```

Comment:
Same here, you can remove FloatReg overloads of Emit(Load|Store) and FloatReg itself.